File: Android/app/src/main/java/com/dijoux/marc/quizz/app/Question.java
package com.dijoux.marc.quizz.app;

public class Question {
    // private String enonce;
    // private String reponse;
    private int enonce;
    private int reponse;
    private boolean difficile;
    private int compteur;

    // public Question(String enonce, String reponse, boolean difficile) {
    public Question(int enonce, int reponse, boolean difficile) {
        this.enonce = enonce;
        this.reponse = reponse;
        this.difficile = difficile;
        this.compteur = 0;
    }

    public int getEnonce() { return enonce; }
    public int getReponse() { return reponse; }
    public boolean estDifficile() { return difficile; }
    public int getCompteur() { return compteur; }
    public void incrCompteur() { compteur++; }
}

File: iOS/Quizz/Question.m
#import "Question.h"

@implementation Question


- (id) initWithTitre:(NSString*)unTitre reponse:(NSString*)uneReponse difficulte:(BOOL)uneDifficulte {
	if (self = [super init]) {
		enonce = [unTitre copy];
		reponse = [uneReponse copy];
        difficulte = uneDifficulte;
		compteur = 0;
	}
	return self;
}


- (NSString*)getEnonce
{
    return enonce;
}

- (void)setEnonce:(NSString *)unEnonce
{
    if (unEnonce != enonce)
    {
        [enonce release];
        enonce = [unEnonce copy];
    }
}

- (NSString*)getReponse
{
    return reponse;
}

- (void)setReponse:(NSString*)uneReponse
{
    if (reponse != uneReponse)
    {
        [reponse release];
        reponse = [uneReponse copy];
    }
}

- (BOOL)getDifficulte
{
    return difficulte;
}

- (void)setDifficulte:(BOOL)uneDifficulte
{
    difficulte = uneDifficulte;
}

- (int)getCompteur
{
    return compteur;
}

- (void)setCompteur:(int)uneValeur
{
    compteur = uneValeur;
}

- (void)dealloc
{
    [enonce release];
    [reponse release];
    [super dealloc];
}

@end
